### PR TITLE
Done: class Student with reading of attributes and changing average grade, and initialization of it.

### PR DESCRIPTION
**Pull request by VeronikaDi**
Hello,
HW 14 done in two files. 
One with class and function and second with initialization of it.
Thank you

### CODE REVIEW

File: lesson_14/class_student.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Student:
+
```
Comment (DartBrovsky) [inline lesson_14/class_student.py:2]: Where is class fields? You need to create fields for further properties assignment.

File: lesson_14/class_student.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, name, last_name, age, average_grade):
```
Comment (DartBrovsky) [inline lesson_14/class_student.py:3]: For constructor it is better to provide variables types.

**Review by DartBrovsky — CHANGES_REQUESTED** [review]
We are good with your homework solution. Please fix my comments and I will approve.

File: lesson_14/class_student.py (hunk 1 above)
Comment (VeronikaDi) [reply lesson_14/class_student.py:2]: What you mean is that I need to add the following:
name: str
last_name: str
age: int
average_grade: float  ?

File: lesson_14/class_student.py (hunk 2 above)
Comment (VeronikaDi) [reply lesson_14/class_student.py:3]: Does this mean I need to write like this:
def __init__(self, name: str, last_name: str, age: int, average_grade: float) ?

**Pull request closed without merging**